libs/feature/search/src/lib/fuzzy-search/fuzzy-search.component.ts

```diff
@@ -41,6 +41,7 @@ export class FuzzySearchComponent {
         JSON.stringify(this.esService.buildAutocompletePayload(query))
       )
       .pipe(
+        take(1),
```

Comment:
useless, we'll discuss about that later.
don't secure it for now, it wasn't before anyway.

Reply:
ok